ADQL.tex

```diff
@@ -1464,17 +1464,20 @@ \subsubsection{Datatype functions}
 \subsubsection{Coordsys}
 \label{sec:geom.coordsys.param}
 
-%TODO check on the list ...
-For historical reasons, the geometry constructors (BOX, CIRCLE, POINT
-and POLYGON) all accept an optional string literal as the first argument.
-This was originally intended to carry
-information on a reference system or other coordinate system metadata.
-As of this version of the specification this parameter has been
-marked as deprecated. Services are permitted to ignore this parameter and
-clients are advised to pass an empty string here. Future versions of this
-specification may remove this parameter from the listed functions.
+For historical reasons, the geometry constructors (\verb:BOX:, \verb:CIRCLE:,
+\verb:POINT: and \verb:POLYGON:) all accept a string literal as the first
+argument, hereafter called the COOSYS argument. Since version 2.1 of the
+specification, this parameter has been made optional and marked as deprecated.
+Future versions of this specification may remove this parameter from the listed
+functions.
+
+This specification makes no guarantees for any semantics for the deprecated
```

Comment:
I think it would be "guarantees on" or perhaps "guarantees about" -- perhaps a native speaker could comment?

Reply:
Probably "guarantees about" would be best

Reply:
Done